brave/browser-laptop
# Don't register a visit if page returns a non-successful HTTP status code

Don't register a visit if page returns a non-successful HTTP status code

Fixes https://github.com/brave/browser-laptop/issues/3759
Auditors:
@mrose17, @bridiver

Test Plan:
1. Visit https://clifton.io
2. Stay on page for a while
3. Confirm time spent is incremented
4. Visit https://clifton.io/404
5. Stay on page for a while
6. Confirm time spent is not incremented
